drivers/hmis/lib/form_data/default/records/ce_event.json

```diff
@@ -0,0 +1,164 @@
+{
+  "item": [
+    {
+      "type": "GROUP",
+      "link_id": "event-group",
+      "item": [
+        {
+          "type": "DATE",
+          "required": true,
+          "link_id": "4.20.1",
+          "text": "Event Date",
+          "mapping": {
+            "field_name": "eventDate"
+          }
+        },
+        {
+          "type": "CHOICE",
+          "required": true,
+          "text": "Event Type",
+          "pick_list_reference": "EventType",
+          "link_id": "4.20.2",
+          "mapping": { "field_name": "event" }
+        },
+        {
+          "type": "CHOICE",
+          "required": false,
+          "text": "Client housed/re-housed in a safe alternative",
+          "pick_list_reference": "NoYesMissing",
+          "link_id": "4.20.A",
+          "mapping": { "field_name": "probSolDivRrResult" },
+          "enable_when": [
+            {
+              "question": "4.20.2",
+              "operator": "EQUAL",
+              "answer_code": "PROBLEM_SOLVING_DIVERSION_RAPID_RESOLUTION_INTERVENTION_OR_SERVICE"
+            }
+          ]
+        },
+        {
+          "type": "CHOICE",
+          "required": false,
+          "text": "Enrolled in Aftercare project",
+          "pick_list_reference": "NoYesMissing",
+          "link_id": "4.20.B",
+          "mapping": { "field_name": "referralCaseManageAfter" },
+          "enable_when": [
+            {
+              "question": "4.20.2",
+              "operator": "EQUAL",
+              "answer_code": "REFERRAL_TO_POST_PLACEMENT_FOLLOW_UP_CASE_MANAGEMENT"
+            }
+          ]
+        },
+        {
+          "type": "TEXT",
+          "required": false,
+          "link_id": "4.20.C",
+          "text": "Location of Crisis Housing or Permanent Housing Referral",
+          "mapping": { "field_name": "locationCrisisOrPhHousing" },
```

Comment:
<img width="554" alt="Screen Shot 2023-08-16 at 8 06 15 PM" src="https://github.com/greenriver/hmis-warehouse/assets/5333982/155c65fd-f512-4913-a1ba-934ad80e4ff6">

Reply:
@awisema , this looks like one place where the "receives CE referrals" flag is used.